tools/src/h5dump/h5dump.c

```diff
@@ -256,7 +256,7 @@ usage(const char *prog)
                    "                          <cred> :: \"(<aws-region>,<access-id>,<access-key>)\"\n");
     PRINTVALSTREAM(rawoutstream,
                    "                          If absent or <cred> -> \"(,,)\", no authentication.\n");
-    PRINTVALSTREAM(rawoutstream, "                          Has no effect is filedriver is not `ros3'.\n");
+    PRINTVALSTREAM(rawoutstream, "                          Has no effect if filedriver is not `ros3'.\n");
     PRINTVALSTREAM(rawoutstream,
```

Comment:
Please change ` to ' in ros3. I don't know why backquote is used.

Reply:
Good catch! In fact why is it different then all other usage, where double quotes are used.

Reply:
This is a common UNIX-y convention. It's used throughout the library.

Reply:
Except it was an outlier here in the usage text.

Reply:
Personally, I hate it because it's harder to type a backtick than a single quote. I think we should get rid of them.

Reply:
Agreed, and I did replace these in a subsequent commit.